test/test_prototype_transforms.py

```diff
@@ -152,3 +153,56 @@ def test_normalize(self, transform, input):
     )
     def test_random_resized_crop(self, transform, input):
         transform(input)
+
+
+class TestRandomHorizontalFlip:
+    def input_tensor(self, dtype=torch.float32):
+        return torch.tensor([[[0, 1], [0, 1]], [[1, 0], [1, 0]]], dtype=dtype)
+
+    def expected_tensor(self, dtype=torch.float32):
+        return torch.tensor([[[1, 0], [1, 0]], [[0, 1], [0, 1]]], dtype=dtype)
+
+    @pytest.mark.parametrize("p", [0.0, 1.0], ids=["p=0", "p=1"])
```

Comment:
1. For builtin types like `int`, `float`, `str`, you don't need to put a manual id. `pytest` handles it for you.

    ```suggestion
        @pytest.mark.parametrize("p", [0.0, 1.0])
    ```

2. IIUC, you want this parametrization on every test in this class. If that is the case, you can simply decorate the class rather than every test individually.